src/Client.cpp

```diff
@@ -19,11 +19,16 @@
 #include <znc/User.h>
 #include <znc/IRCNetwork.h>
 #include <znc/Query.h>
+#include <algorithm>
 
 using std::set;
 using std::map;
 using std::vector;
 
+static const std::map<CString, std::function<void(CClient* pClient, bool bVal)>> g_mNotifyableCaps = {
```

Comment:
1. ```error: constexpr variable cannot have non-literal type 'const std::map<T1, T2>'```
2. now it is a member so it became ```m_mCoreCaps``` or should it also have the map,string,tuple,bool,func types as in ```m_mstbfCoreCaps```? :)
3. done